jenkins/defaults.yaml

```diff
@@ -12,9 +12,9 @@ jenkins:
   home: /var/lib/jenkins
   java_args: -Djava.awt.headless=true
   java_executable: /usr/bin/java
-  jenkins_args:
+  jenkins_args: ''
   max_open_files: 32768
-  umask: 027
+  umask: '027'
```

Comment:
does this need to be changed?

in the file `jenkins/files/Debian/jenkins.conf`, the default (commented) version:

```
 57 # UMASK=027
 58 UMASK={{ jenkins.umask }}
```